components/NavBar.tsx

```diff
@@ -0,0 +1,70 @@
+"use client";
+import React from "react";
+import Image from "next/image";
+import AppBar from "@mui/material/AppBar";
+import Toolbar from "@mui/material/Toolbar";
+import Typography from "@mui/material/Typography";
+import { Stack } from "@mui/material";
+import Navigation from "@/constants/Navigation";
+import NavBarButtonVisita from "@/components/ui/NavBarButtonVisita";
+import { dosis } from "@/fonts/font";
+import clinic from "@/public/clinic.jpg";
```

Comment:
Please avoid jpg images in production. For now, it's OK.
And the path is @/public/images/imagefile.png for example.

Public folder structure is:

public/audios
public/icons
public/images
public/videos, etc.